plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -96,7 +96,11 @@ test.describe( 'Shopper → Tax', () => {
 
 		// Hovering over the mini cart should not change the label,
 		// see https://github.com/woocommerce/woocommerce/issues/43691
-		page.getByTestId( 'mini-cart' ).getByLabel( '1 item in cart' ).hover();
+		await page
+			.getByTestId( 'mini-cart' )
+			.getByLabel( '1 item in cart' )
+			.dispatchEvent( 'mouseover' );
```

Comment:
Why did this work and not `hover`?

Reply:
To be fair, I do not have an explanation for that. Both @opr and I were also surprised by this, as we expected that `hover` should work. We ran several tests, that proved that `dispatchEvent( 'mouseover' )` behaves differently from `hover` in Playwright.